api/types/constants.go

```diff
@@ -440,6 +440,12 @@ const (
 	// that the resource originates from.
 	OriginLabel = TeleportNamespace + "/origin"
 
+	// CreatorTypeLabel is a resource metadata label name used to identify the creator type.
+	CreatorTypeLabel = TeleportNamespace + "/creator_type"
+
+	// CreatorLabel is a resource metadata label name used to identify the creator name.
+	CreatorLabel = TeleportNamespace + "/creator"
```

Comment:
Why not just call it "cluster"? Creator is quite confusing.
```suggestion
	ClusterLabel = TeleportNamespace + "/cluster"
```

Reply:
Replacing CreatorLabel with ClusterLabel 👍